pkg/diffs/diffs.go

```diff
@@ -166,6 +195,15 @@ func GetPresubmitsForCiopConfigs(prowConfig *prowconfig.Config, ciopConfigs conf
 				}
 				if config.IsCiopConfigCM(env.ValueFrom.ConfigMapKeyRef.Name) {
 					if _, ok := ciopConfigs[env.ValueFrom.ConfigMapKeyRef.Key]; ok {
+
+						s := strings.Split(job.Name, "-")
+						testName := s[len(s)-1]
```

Comment:
I'm afraid this won't be enough, the test names may contain a `-` (`e2e-aws` and many more...), so the last `-`-separated fragment won't always be the test name...

Reply:
Nice catch! I will address this. Thanks!

Reply:
addressed